playground/src/screens/OptionsScreen.js

```diff
@@ -190,6 +194,14 @@ class OptionsScreen extends Component {
     });
   }
 
+  onClickShowOverlay(passThroughTouches) {
+    Navigation.showOverlay({
+      component: {
+        name: 'navigation.playground.CustomDialog'
+      }
+    }, { passThroughTouches });
```

Comment:
passThroughTouches should be part of `options` I think

Reply:
done

Reply:
and let's rename that to `interceptTouches`